tasks/Recall Images.md

```diff
@@ -43,7 +43,6 @@ Optimization, and Shapes
 
 ## Instructions
 Group Instructions: Welcome everyone! My name is ___, thanks for joining us for today’s session. The plan for today is for you to work on a set of tasks very similar to those that you completed in the individual, online assessments. Today we’ll be working in person, in groups of three people.
-Figure 1: Overview of Experiment Notes: this figure describes the experimental design from an individual participant’s perspective. Tasks are described in Section 2.2. Task batteries represent unique sequences of tasks. Participants never see the same exact task more than once. Lab visits involve 6, 9 or 12 participants, who were randomly allocated to groups of 3 people - see Sections 2.3 and 2.4 for details. Battery A Problem Solving (Optimization A; Memory A; Shapes A)Personality (Big5)Emotional perception (RMET)Participants complete tests online (usually at home)Phase 1: Individual testingPhase 2: Group testingLab Visit 1 1st group2nd group Battery B Optimization B Memory B Shapes B Battery C Optimization C Memory C Shapes C Battery D Optimization D Memory D Shapes D Battery E Optimization E Memory E Shapes E CryptographyLab Visit 23rd group 4th group
```

Comment:
Just a reminder, these things are not new lines to markdown. They would need new lines between each paragraph to look appear as actual new lines in the rendered file. 

e.g., if you look here → https://github.com/Watts-Lab/task-mapping/blob/621ce1ab6c3a4fe74329d375dbcad566c8e338fe/tasks/Recall%20Images.md#instructions 

its all together. Is that what you want?

Reply:
I think I resolved this- but let me know if I'm not understanding your comment.

Reply:
Yep looks good! 

<img width="1097" alt="Screen Shot 2021-07-22 at 12 00 44" src="https://user-images.githubusercontent.com/673171/126670798-0e57179f-eba6-4887-85f3-863174570b56.png">